iocage/lib/Jail.py

```diff
@@ -1633,53 +1636,29 @@ def _limit_resources(self) -> None:
             self.logger.verbose("Resource limits disabled")
             return
 
-        for key, limit, action in map(
-            lambda name: (
-                (name, ) + self._get_resource_limit(name)  # noqa: T484
-            ),
-            self._resource_limit_config_keys
-        ):
-
-            if (limit is None) and (action is None):
-                # this resource is not limited (limit disabled)
+        for key in iocage.lib.Config.Jail.Properties.ResourceLimit.properties:
+            try:
+                rlimit_prop = self.config[key]
+            except KeyError:
                 continue
```

Comment:
logging happens on set? right?
so, when loading a corrupt legacy config, we don't know… correct?

Reply:
We do because parsing the rule fails. The next changes will call set to initialize the value anyway, so that it has to pass the parser once.

Reply:
sorry, i meant: we don't tell the user